src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculator {
```

Comment:
let's implement AreaCalculator interface in the Figure class and remove implementation from here and from all other classes that extend the Figure class